vnext/ReactWindowsCore/ReactWindowsCore.vcxproj

```diff
@@ -87,6 +86,7 @@
         $(ReactNativeWindowsDir)include\ReactWindowsCore;
         $(ReactNativeDir)\ReactCommon;
         $(JSI_Source);
+        $(ReactNativeWindowsDir)JSI\Shared;
```

Comment:
>$(ReactNativeWindowsDir)JSI\Shared; [](start = 8, length = 35)

I wonder why do we need to add it because the the JSI.Shared.vcxitems object already adds this path:

<AdditionalIncludeDirectories>%(AdditionalIncludeDirectories);$(MSBuildThisFileDirectory)</AdditionalIncludeDirectories>

Reply:
I don't believe the property is transitive in MSBuild. If I remove this line I'm getting compilation errors from JSI\Shared headers not found.

---
In reply to: [387946887](https://github.com/microsoft/react-native-windows/pull/4234#discussion_r387946887) [](ancestors = 387946887)